README.md

```diff
@@ -443,6 +443,7 @@ Name | Website | Region
 [PeopleDoc](/company-profiles/peopledoc.md) | https://www.people-doc.com | Europe
 [Percona](/company-profiles/percona.md) | https://www.percona.com | Worldwide
 [Pex](/company-profiles/pex.md) | https://pex.com | Worldwide
+[Pexip](/company-profiles/pexip.md) | https://pexip.com / https://dev.pexip.com | Europe
```

Comment:
Sorry @Niladev the build fails if there is more than one URL here - can you choose one and remove the other, please?